docs/_data/config-portland-2018.yaml

```diff
@@ -28,7 +28,6 @@ date: # how do we handle these? human readable would be nice.
   short: May 6-8, 2018
   tickets_live: "January 2018"
   month: May
```

Comment:
Do we want to try to add some metadata that would let us fix this? https://github.com/writethedocs/www/pull/1125/files#diff-66dfabd3b4a41d17c53d19ac40e964c6R84